src/java.desktop/share/classes/javax/swing/plaf/metal/MetalLookAndFeel.java

```diff
@@ -272,6 +272,7 @@ protected void initClassDefaults(UIDefaults table)
               "FileChooserUI", metalPackageName + "MetalFileChooserUI",
             "InternalFrameUI", metalPackageName + "MetalInternalFrameUI",
                     "LabelUI", metalPackageName + "MetalLabelUI",
+                  "MenuBarUI", metalPackageName + "MetalMenuBarUI",
        "PopupMenuSeparatorUI", metalPackageName + "MetalPopupMenuSeparatorUI",
```

Comment:
I think it is installed by the Ocean theme, which is used by the Metal L&F by default.

Reply:
Spec updated